query-planner-js/src/schema-helper/transformSchema.ts

```diff
@@ -65,7 +65,7 @@ export function transformSchema(
     query: replaceMaybeType(schemaConfig.query),
     mutation: replaceMaybeType(schemaConfig.mutation),
     subscription: replaceMaybeType(schemaConfig.subscription),
-    directives: replaceDirectives(schemaConfig.directives)
+    directives: replaceDirectives(schemaConfig.directives) as GraphQLDirective[],
```

Comment:
I'm surprised this works across both versions, wasn't this giving us trouble as one of the `Const...` node types?

Reply:
It's just casting away the readonly property. The issue we had before was that we had a function returning a `ConstValueNode` which wasn't in the same type hierarchy as `ValueNode`